R/clustering.R

```diff
@@ -747,7 +736,7 @@ RunLeiden <- function(
     object = input,
     partition_type = partition.type,
     initial_membership = initial.membership,
-    weights = weights,
+    weights = NULL,
```

Comment:
```suggestion
    weights = NULL,
```

Remove in next release (deprecate for now).